client/app/components/users/UserEdit.jsx

```diff
@@ -176,7 +176,7 @@ export default class UserEdit extends React.Component {
     const { user, regeneratingApiKey } = this.state;
 
     return (
-      <Form layout="vertical">
+      <Form className="hide-in-percy" layout="vertical">
```

Comment:
I think it's important to hide only the precise offending content `<InputWithCopy>`.